app/models/question.rb

```diff
@@ -227,23 +214,24 @@ def update_conditions(param_conditions)
   end
 
 
-  def saveCondition(value, option_id, remove_id = nil)
-    c = self.question_options.find(option_id).conditions.build
-    c.action_type = value['action_type']
-    c.remove_question_id = remove_id
+  def saveCondition(value)
+    c = self.conditions.build
+    c.action_type = value["action_type"]
     c.number = value['number']
-    c.webhook_data = get_webhook(value)
+    c.option_list = value["question_option"]
+    if value["action_type"] == "remove"
+      c.remove_data = value["remove_question_id"]
+    else
+      c.webhook_data = {
+        name: ActionController::Base.helpers.sanitize(value['webhook-name']),
```

Comment:
maybe better to just do the sanitization at the controller before passing the hash to this method since the helper is meant to be used in that context